lib/DDG/Spice/Goodreads.pm

```diff
@@ -0,0 +1,57 @@
+package DDG::Spice::Goodreads;
+
+use strict;
+use DDG::Spice;
+use URI::Escape;
+use Data::Dump qw(dump);
+
+# Caching
+spice is_cached => 1;
+
+# Goodreads API endpoint
+my $api_key = '{{ENV{DDG_SPICE_GOODREADS_APIKEY}}}';
+my $book_search_endpoint = uri_escape('https://www.goodreads.com/search/index.xml?key=') . $api_key . uri_escape('&search[field]=') . '$1' . uri_escape('&q=') . '$2';
+
+spice to => 'https://duckduckgo.com/x.js?u=' . $book_search_endpoint;
+spice wrap_jsonp_callback => 1;
+
+# Alternative end points to fetch book description and image.
+spice alt_to => {
+  get_book_details => {
+    to => 'https://duckduckgo.com/x.js?u=' . uri_escape("https://www.goodreads.com/book/show/") . '$1' . uri_escape("?format=xml&key=") . $api_key,
+    wrap_jsonp_callback => 1
```

Comment:
what exactly are you pointing out be unnecessary here ?
The padding callback statement? 'wrap_jsonp_callback => 1'